.prettierignore

```diff
@@ -0,0 +1 @@
+src/content/blog/summer-of-codegen.mdx
```

Comment:
Whenever I saved the code prettier ran and broke stuff, so I had to disable it. Open to other options here